install-github.R

```diff
@@ -3784,7 +3830,25 @@ safe_install_packages <- function(...) {
   )
 }
 
-safe_build_package <- function(pkgdir, build_opts, dest_path, quiet, use_pkgbuild = !is_standalone() && pkg_installed("pkgbuild")) {
+normalize_build_opts <- function(build_opts, build_manual, build_vignettes) {
+  if (!isTRUE(build_manual)) {
+    build_opts <- c(build_opts, "--no-manual")
```

Comment:
Just for parallelism with `setdiff()`, should this `c()` and the one for `build_vignettes` be `union()` and then you could ditch the `unique()`?

Reply:
That is a good suggestion, yeah